server-phoenix/lib/helios/announcement.ex

```diff
@@ -1,18 +1,31 @@
 defmodule Helios.Announcement do
   use Ecto.Schema
   import Ecto.Changeset
-  alias Helios.Announcement
+  import Ecto.Query
+  alias Helios.Location
 
   @primary_key {:id, :id, autogenerate: true}
   schema "announcements" do
-    field :publish_on, :utc_datetime
+    field :publish_on, :utc_datetime, presence: true
     field :message, :string
-    field :people, :string
+    field :people, :string, presence: true
     field :company, :string
-    field :announcement_id, :string
-    field :created_at, :utc_datetime
-    field :updated_at, :utc_datetime
+    field :announcement_id, :string, presence: true
+    field :created_at, :utc_datetime, presence: true
+    field :updated_at, :utc_datetime, presence: true
```

Comment:
It looks like you squashed these changes into the scopes commit instead of the schema commit.